inc/compatibilities/elementor_builder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Class Optml_elementor_builder
+ *
+ * @reason Adding selectors for background lazyload
+ */
+class Optml_elementor_builder extends Optml_compatibility {
+
+
+	/**
+	 * Should we load the integration logic.
+	 *
+	 * @return bool Should we load.
+	 */
+	function should_load() {
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+		return is_plugin_active( 'elementor/elementor.php' );
+	}
+
+	/**
+	 * Register integration details.
+	 */
+	public function register() {
+		add_filter(
+			'optml_lazyload_bg_selectors',
+			function ( $all_watchers ) {
+				$all_watchers = array_merge( $all_watchers, ['.elementor-widget-container', '.elementor-background-overlay', '.div-with-background > .elementor-column-wrap'] );
```

Comment:
@GrigoreMihai `.div-with-background` i dont think it's a standard elementor class, and .elementor-background-overlay is already used here https://github.com/Codeinwp/optimole-wp/blob/master/inc/lazyload_replacer.php#L86

Reply:
ok